media-libs/dav1d/dav1d-0.5.2.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="A new AV1 cross-platform decoder"
+DESCRIPTION="dav1d is a new AV1 cross-platform decoder, open-source, \
+and focused on speed and correctness. It supports all features from AV1, \
+including all subsampling and bit-depth parameters."
+HOMEPAGE="https://code.videolan.org/videolan/dav1d"
+COPYRIGHT="2018-2019, VideoLAN and dav1d authors"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://code.videolan.org/videolan/dav1d/-/archive/$portVersion/dav1d-$portVersion.tar.bz2"
+CHECKSUM_SHA256="1780199d71e62a82ca1bfca2d71da5c599c7e1edc68c72f6fc3c0c25ce1880b7"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64 ?arm ?ppc"
```

Comment:
This doesn't build for x86_gcc2 (nasm's latest version doesn't also), so add SECONDARY_ARCHITECTURES and $secondaryArchSuffix in the needed places :)

Reply:
Does `$secondaryArchSuffix` affect build for arm and ppc?

Reply:
nope, those are primary targets

Reply:
Thanks.